commands/acl-cat/index.html

```diff
@@ -0,0 +1,5 @@
+---
+layout: command
+title: acl-cat
+description: acl-cat.md
+---
```

Comment:
Are these files generated? In that case, I don't think they should be checked in.

When all is in place, then when valkey-doc is updated, the website should preferably be updated automatically without any changes to the valky-website repo. This should include the addition of new commands, without adding files like this to the website repo.

Reply:
@zuiderkwast They are generated and I actually agree, the PR body `Why so many files?` and `Submodules` sections I talk about both points.

Right now, the build infrastructure for the site is a black box to me. Ideally, we'd bring these in with a GitHub action, but I think the site is built by some other means. 

But if you we merge this, it will work today without anything.

Reply:
Right, I didn't read the description properly. :)

Yeah, let's merge this and we'll remove those files again in the future.

Reply:
I took a 10 minute pass to investigate this, and wasn't able to figure it out quickly. We nee to de-blackbox it and add documentation, but agree about not blocking on this.